components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.geo-devices/public/js/geo_stationary.js

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+function initStationaryAlert() {
+    // var serverUrl = "/portal/store/carbon.super/fs/gadget/geo-dashboard/controllers/get_alerts.jag?executionPlanType=Stationery&deviceId=" + deviceId;
+    var serverUrl = "/api/device-mgt/v1.0/geo-services/alerts/Stationery";
+
+    $(".removeGeoFence").tooltip();
+    $("#stationary-alert-table > tbody").empty();
+    invokerUtil.get(serverUrl, function (response) {
+        if (response == "[]") {
+            $(".fence-not-exist").show();
+            $("#stationary-alert-table").hide();
+        }
```

Comment:
this `if` check can be safely removed by making `else if (response) {` -> `if (response) {` in line 30.

Reply:
Fixed c653051